sycl/test/inline-asm/negative_tests/asm_bad_opcode.cpp

```diff
@@ -0,0 +1,43 @@
+// UNSUPPORTED: cuda
+// REQUIRES: gpu,linux
+// RUN: %clangxx -fsycl %s -DINLINE_ASM -o %t.out
```

Comment:
Why do you need this INLIONE_ASM macro since you seem to always define it?

Reply:
It remained from positive tests, I will delete it.